Creación del documento solitado en el issue #93

wpwapuu opened the pull request:
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam condimentum, elit et efficitur euismod, metus elit malesuada est, aliquam egestas ligula sapien nec est. Phasellus pellentesque mi et turpis egestas, in aliquam libero suscipit. Donec dignissim nisl sit amet est mollis gravida at quis ex. Donec sed imperdiet lorem, eu ultricies enim. Nullam eu dictum nisl. Morbi malesuada elit eu erat tempor malesuada. In non pulvinar est, eget tempus ligula.

<img width="556" alt="CleanShot 2022-12-06 at 11 07 41@2x" src="https://user-images.githubusercontent.com/119888230/205881710-4e27ad9b-ce30-442b-9ed8-5725228c3e34.png">

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam condimentum, elit et efficitur euismod, metus elit malesuada est, aliquam egestas ligula sapien nec est. Phasellus pellentesque mi et turpis egestas, in aliquam libero suscipit. Donec dignissim nisl sit amet est mollis gravida at quis ex. Donec sed imperdiet lorem, eu ultricies enim. Nullam eu dictum nisl. Morbi malesuada elit eu erat tempor malesuada. In non pulvinar est, eget tempus ligula.

jesusyesares committed:
Creada carpeta de prueba para vídeotutorial

wpwapuu committed:
Creado el documento en latín

Creado el documento en latín que se pedía en el issue https://github.com/WPES/spain-handbook/issues/93.

Tan solo he copiado y pegado lo que había en el documento de google drive.

wpwapuu committed:
He añadido saltos de línea y unas negritas

wpwapuu committed:
Texto que faltaba añadido

He añadido este texo porque creo que es muy explicativo sobre cómo funciona la cosa realmente en el nuevo orden mundial de la vida.

jesusyesares commented on ES-handbook-marketing/manual-comunicacion-wordcamps/anexos/documentos-en-latin/documento-en-latin.md:
```diff
@@ -1,7 +1,9 @@
-Pellentesque auctor faucibus turpis, id ullamcorper eros molestie eu. Etiam condimentum porta tellus, vitae dignissim sem condimentum id. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Aliquam congue mattis tristique. Sed sed hendrerit est. Morbi vitae turpis purus. Aliquam molestie nulla arcu, eleifend convallis sem mollis sit amet. Aenean dui libero, iac ...
+
+In sit amet sapien magna. Aliquam imperdiet rhoncus convallis. Suspendisse finibus, quam at ullamcorper lobortis, nibh nibh venenatis neque, in dictum turpis augue nec quam. Integer lorem lorem, facilisis ac sollicitudin in, dapibus ac quam. Nam aliquam dui sit amet aliquet rhoncus. Aliquam egestas sodales nulla quis tincidunt. Mauris nec neque vitae nisi porttitor egestas vehicula sit amet magna ...
+Mauris tincidunt, **magna** vitae posuere laoreet, eros lorem dapibus mauris, nec cursus purus felis eget nunc. Cras ullamcorper quam leo, at scelerisque urna vulputate id. Nulla maximus vel nisi et laoreet. Vestibulum vitae dapibus elit, ac lacinia dui. Nulla in tortor orci. Nulla arcu elit, eleifend eget congue ut, malesuada et magna. Suspendisse sit amet urna dapibus, feugiat dui at, aliquet e ...
```
Esta palabra no debería ir en negrita. Por favor, cámbiala a cursiva.

wpwapuu replied:
Lo hice por este motivo

jesusyesares commented on ES-handbook-marketing/manual-comunicacion-wordcamps/anexos/documentos-en-latin/documento-en-latin.md:
```diff
@@ -6,4 +6,6 @@ Morbi nec lacinia turpis. Suspendisse sapien nisl, sagittis eu tristique sit ame
 In sit amet sapien magna. Aliquam imperdiet rhoncus convallis. Suspendisse finibus, quam at ullamcorper lobortis, nibh nibh venenatis neque, in dictum turpis augue nec quam. Integer lorem lorem, facilisis ac sollicitudin in, dapibus ac quam. Nam aliquam dui sit amet aliquet rhoncus. Aliquam egestas sodales nulla quis tincidunt. Mauris nec neque vitae nisi porttitor egestas vehicula sit amet magna ...
 Mauris tincidunt, **magna** vitae posuere laoreet, eros lorem dapibus mauris, nec cursus purus felis eget nunc. Cras ullamcorper quam leo, at scelerisque urna vulputate id. Nulla maximus vel nisi et laoreet. Vestibulum vitae dapibus elit, ac lacinia dui. Nulla in tortor orci. Nulla arcu elit, eleifend eget congue ut, malesuada et magna. Suspendisse sit amet urna dapibus, feugiat dui at, aliquet e ...
 
+Aquí añadiremos un texto que hacía falta añadir porque es muy importante para la humanidad.
```
Me parece una buena idea, pero creo que no está del todo claro, por lo que te pido que lo documentes un poco mejor, e incluso añadas alguna captura de pantalla.

jesusyesares reviewed (commented):
Por favor, revisa los pequeños cambios que te solicito.

wpwapuu committed:
Cambiada la palabra magna a cursiva

wpwapuu committed:
Añadido el texto con captura de pantalla

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam condimentum, elit et efficitur euismod, metus elit malesuada est, aliquam egestas ligula sapien nec est. Phasellus pellentesque mi et turpis egestas, in aliquam libero suscipit. Donec dignissim nisl sit amet est mollis gravida at quis ex. Donec sed imperdiet lorem, eu ultricies enim. Nullam eu dictum nisl. Morbi malesuada elit eu erat tempor malesuada. In non pulvinar est, eget tempus ligula.

wpwapuu reviewed (commented):
Hola Jesús, ya he hecho los cambios que me pedías. 

He tenido un problemilla en no se qué, pero lo he resuelto.

jesusyesares reviewed (approved):
Muchas gracias. Ya he aprobado tus cambios.
